hermes-common/src/main/java/pl/allegro/tech/hermes/common/message/converter/AvroToJsonConverter.java

```diff
@@ -0,0 +1,49 @@
+package pl.allegro.tech.hermes.common.message.converter;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DecoderFactory;
+import org.apache.avro.io.EncoderFactory;
+import org.apache.avro.io.JsonEncoder;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class AvroToJsonConverter {
+
+    private final GenericDatumReader<GenericRecord> reader;
+    private final GenericDatumWriter<GenericRecord> writer;
+    private final Schema schema;
+
+    public AvroToJsonConverter(Schema schema) {
+        this.schema = schema;
+        this.reader = new GenericDatumReader<>(schema);
+        this.writer = new GenericDatumWriter<>(schema);
+    }
+
+    public byte [] convert(byte [] data) {
+        try {
+            BinaryDecoder binaryDecoder = DecoderFactory.get().binaryDecoder(data, null);
```

Comment:
avro 'reuse' is helpful when you deal with a stream of data. For example when you read data from file or some output stream. In multithreaded env like this it may cause problems - for example more than one thread can work on same byte array. I am curious what is latency for this methods.